### PR TITLE
docs(external): add typing and docstrings for mrvi, remove laplace scaling

### PR DESCRIPTION
**Pull request by martinkim0**

### CODE REVIEW

File: src/scvi/external/mrvi/_module.py [hunk 1]
```diff
@@ -202,7 +286,55 @@ def __call__(
...
+    u_prior_mixture
+        TODO
+    u_prior_mixture_k
+        TODO
+    learn_z_u_prior_scale
+        TODO
+    laplace_scale
+        TODO
+    scale_observations
+        TODO
```
Comment (martinkim0) [inline src/scvi/external/mrvi/_module.py:326]: Would love to get input on these

File: src/scvi/external/mrvi/_module.py [hunk 2]
```diff
@@ -110,6 +140,40 @@ def __call__(
...
+    n_heads
+        Number of heads in the attention block.
+    dropout_rate
+        Dropout rate.
+    stop_gradients
+        Whether to stop gradients to ``u``.
+    stop_gradients_mlp
+        Whether to stop gradients to the MLP in the attention block.
+    use_map
+        TODO
```
Comment (martinkim0) [inline src/scvi/external/mrvi/_module.py:166]: This one too

File: src/scvi/external/mrvi/_module.py (hunk 1 above)
Comment (justjhong) [reply src/scvi/external/mrvi/_module.py:325]: z_u_prior
    Whether to place a Gaussian prior on ``z`` given ``u``.
z_u_prior_scale
    Natural log of the scale parameter of the Gaussian prior placed on ``z`` given ``u``. Only applies of ``learn_z_u_prior_scale=False``.
u_prior_scale
    Natural log of the scale parameter of the Gaussian prior placed on ``u``. If ``u_prior_mixture=True``, this scale applies to each mixture component distribution.
u_prior_mixture
    Whether to use a mixture of Gaussians prior for ``u``.
u_prior_mixture_k
    Number of mixture components to use for the mixture of Gaussians prior on ``u``.
learn_z_u_prior_scale
    Whether to learn the scale parameter of the prior distribution of ``z`` given ``u``.
scale_observations
    Whether to scale the loss associated with each observation by the total number of observations linked to the associated sample.
    

Let's remove laplace scale (it was used for the old linear model) and all traces of "As"

**Review by justjhong — COMMENTED** [review]
Thank you so much for this!

**Comment by martinkim0** [comment]
Thanks @justjhong! Just removed instances of laplace and As in the code.

**Pull request merged**